simplq/src/styles/adminPage.module.scss

```diff
@@ -223,21 +233,23 @@
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
-  margin-bottom: 3rem;
+  margin: 3rem 10rem;
 }
 
 .navbar {
   width: 100%;
   height: 4rem;
   background-color: #fff;
   border-bottom: 0.2rem solid #f1f1f1;
-  padding: 0 3rem;
+  padding: 0 11rem;
```

Comment:
I am having trouble making sense of these margins, I'll have to pull the code locally and see, before I can approve these changes.

Can you pull our latest code and fix conflicts?